samples/testing-frameworks/appium/client-side/java/src/test/java/com/testdroid/appium/ios/sample/IosAppiumExampleTest.java

```diff
@@ -26,13 +26,13 @@ public void mainPageTest() {
         wd.manage().timeouts().implicitlyWait(60, TimeUnit.SECONDS);
         wd.findElement(By.xpath("//*[contains(@name, 'answer1')]")).click();
         WebElement element = wd.findElement(By.xpath("//*[contains(@name, 'userName')]"));
-        takeScreenshot("answer1_element_clicked");
+//        takeScreenshot("answer1_element_clicked");
```

Comment:
Are we no longer taking screenshots in test or you left it like that for testing purposes?